ml-agents/mlagents/trainers/lesson_controller.py

```diff
@@ -0,0 +1,122 @@
+import os
+import yaml
+import math
+
+from .exception import LessonControllerError
+
+import logging
+
+logger = logging.getLogger("mlagents.trainers")
+
+
+class LessonController(object):
```

Comment:
nit: in python3 you don't need to explicitly inherit from object. just `class LessonController:` is fine.